hw_decorator.py

```diff
@@ -0,0 +1,99 @@
+import functools
+
+
+# ЗАДАЧА-1
+# Написать свой декоратор который будет проверять остаток от деления числа 100 на результат работы функции ниже.
+# Если остаток от деления = 0, вывести сообщение "We are OK!», иначе «Bad news guys, we got {}» остаток от деления.
+
+def divide_by_hundred(my_func):
+    """The decorator function which checks input for a valid type (int or float) and then checks if the result
+        of my_func is divided by 100 without rest or not."""
```

Comment:
In fact, there was a task to create a function that checks if 100 can be divided by the result of the function. Still, it doesn't   matter :)